apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/TransformerTestUtils.java

```diff
@@ -746,6 +746,27 @@ static void assertIdentifierExists(
             .anyMatch(i -> expectedId.equals(i.getValue())));
   }
 
+  /**
+   * FIXME change name of this and related methods to assertHasExtensionCoding(...)
```

Comment:
chris, did you put this FIXME in? if not, then maybe either remove and fix test names?

Reply:
Took out